incubator/binding-mqtt-kafka/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/kafka/internal/stream/MqttKafkaSessionFactory.java

```diff
@@ -721,9 +972,7 @@ private void onKafkaBegin(
             state = MqttKafkaState.openingReply(state);
 
             assert replyAck <= replySeq;
-
-            delegate.doMqttBegin(traceId, authorization, affinity);
-            doKafkaWindow(traceId, authorization, 0, 0);
+            doKafkaWindow(traceId, authorization, 0, 0, 0);
```

Comment:
Is it necessary to send this `kafka` reply window proactively, or can we send it in reaction to `mqtt` reply window instead?